src/components/StoreMigrator/index.tsx

```diff
@@ -0,0 +1,132 @@
+import { ReactElement, useEffect, useState } from 'react'
+import { useDispatch } from 'react-redux'
+import { AddressBookState } from 'src/logic/addressBook/model/addressBook'
+
+import { addressBookMigrate } from 'src/logic/addressBook/store/actions'
+import { Errors, trackError } from 'src/logic/exceptions/CodedException'
+import { saveMigratedKeyToStorage } from 'src/utils/storage'
+import allNetworks from 'src/config/networks'
+
+type MigrationMessageEvent = MessageEvent & {
+  data: { payload: string }
+}
+
+const MIGRATION_KEY = 'SAFE__subdomainsMigrated'
+const IFRAME_PATH = '/migrate-local-storage.html'
+const IFRAME_NAME = 'migrationIframe'
+
+// Take all networks except for local
+const { local, ...migratedNetworks } = allNetworks
```

Comment:
Need to also filter out mainnet if we decide to keep the unified app on the same domain as mainnet.